src/Combinatorics/Matroids/ChowRings.jl

```diff
@@ -0,0 +1,211 @@
+export chow_ring, augmented_chow_ring, select
+
+@doc Markdown.doc"""
+The Chow ring of a matroid, optional also with the simplical generators.
+
+See AHK18 (@cite) and BES21 (@cite) 
+
+
+# Examples
+The following computes the chow ring of the Fano matroid.
+```jldoctest
+julia> M = fano_matroid();
+
+julia> R = chow_ring(M);
+
+julia> R[1]*R[8]
+-x_{3,4,7}^2
+```
+
+# Examples
+The following computes the chow ring of the Fano matroid including variables for the simplicial generators.
+```jldoctest
+julia> M = fano_matroid();
+
+julia> R = chow_ring(M, extended=true);
+
+julia> f = R[22] + R[8] - R[29]
+x_{1,2,3} + h_{1,2,3} - h_{1,2,3,4,5,6,7}
+
+julia> f==0
+true
+```
+"""
+function chow_ring(M::Matroid; extended=false)
+        Flats = flats(M)
```

Comment:
This is a minor nitpick, and nothing we'll hold a merge over, but I'd still appreciate if you'd consider it:
It seems you are indenting with 8 spaces -- we are (annoyingly) not very consistent with indentation, with people using 2, 3, 4 spaces -- but never 8. Please pick one of the lower indent level.

Reply:
Done in combination with replacing tabs